app/queries/occupation_standard_query.rb

```diff
@@ -0,0 +1,16 @@
+class OccupationStandardQuery
+  ATTRIBUTES = %i[
+    search_term
+  ]
+
+  # Container is a wrapper for the params involved in the search
+
+  Container = Struct.new(*ATTRIBUTES, keyword_init: true)
```

Comment:
What is `keyword_init` for?

Reply:
@littleforest this allows initialize an struct with a list of keywords

https://www.bigbinary.com/blog/ruby-2-5-allows-creating-structs-with-keyword-arguments